src/assets/translations/en/main.json

```diff
@@ -974,7 +974,10 @@
       "unsafeQuote": "This amount is below the recommended minimum for this pair (%{recommendedMin} %{symbol}). This could cause your trade to fail or loss of funds.",
       "rateLimitExceeded": "Rate limit exceeded, try again later",
       "maxSlippageExceededWithExpectedSlippage": "The maximum allowed slippage tolerance for this trade has been exceeded during simulation. Expected slippage: %{expectedSlippage}%. Try again with a higher slipppage tolerance.",
-      "executionRevertedWithExpectedSlippage": "Execution reverted during execution. This may be due to insufficient slippage tolerance. Expected slippage: %{expectedSlippage}%. Try again with a higher slipppage tolerance."
+      "executionRevertedWithExpectedSlippage": "Execution reverted during execution. This may be due to insufficient slippage tolerance. Expected slippage: %{expectedSlippage}%. Try again with a higher slipppage tolerance.",
+      "slippageExceeded": "Slippage tolerance exceeded.",
```

Comment:
```suggestion
      "slippageExceeded": "Slippage tolerance exceeded.",
```

Reply:
Done in [389da60](https://github.com/shapeshift/web/pull/8433/commits/389da60661cd195083f5751a43bb80be8b977aa9)